USAGE.md

```diff
@@ -189,6 +216,10 @@ Now you should be able to use the protonvpn command from anywhere in the system
 
     `sudo unlink /usr/local/bin/protonvpn`
 
+## Initialization
+
+Before you can use ProtonVPN-CLI, you need to initialize it. Run `sudo protonvpn init` and follow the prompts on the screen.
+
```

Comment:
Why this is been moved after Uninstall section? At this point the full example of installation on Ubuntu seems a bit out of order, because it describe the initialization, which described only afterwards.

Reply:
@kaplun I moved this section here because it is universal to all installation methods. Previously it was wrapped up with what I renamed the "Installing from PyPI" section, so it looked like you only needed to initialize if you installed from PyPI.

I'm happy to order this another way that makes more sense.

Reply:
Yeah it makes sense indeed. Maybe we can move the section with screenshots from Ubuntu to follow this new dedicated section.

Reply:
I.e. for "Now you have access to the protonvpn command.[...]" to the last "Connect to ProtonVPN" section. Unfortunately the screenshots have an Ubuntu prompt, but I don't think it's a big deal :-)
What do you think?

Reply:
I don't think Ubuntu screenshots are an issue. Someone looking for a command-line tool probably knows how to use their package manager, and wants to copy+paste a line in their terminal.

If you wanted to go above and beyond, there is [Asciicinema](https://asciinema.org/) but it is out of scope for this PR. :slightly_smiling_face:

Reply:
I'll rearrange after merge.